spec/std/socket_spec.cr

```diff
@@ -289,6 +342,19 @@ describe UNIXServer do
       end
     end
 
+    it "returns an abstract client UNIXSocket for abstract server" do
```

Comment:
I know I saw that too, but as there was already 2 redundant tests for non-abstract sockets:
- describe "accept" > it "returns the client UNIXSocket"
- describe "accept?" > it "returns the client UNIXSocket"

I wanted to have some coherence and keep 2 redundant tests for abstract sockets... So I will remove the `accept?` test, but should I remove also the `accept?` test for non-abstract socket, which is redundant too ?